spec/std/openssl/ssl/socket_spec.cr

```diff
@@ -15,7 +15,9 @@ describe OpenSSL::SSL::Socket do
       end
 
       client = server.accept
-      client.cipher.should contain "RSA"
+      cipher = client.cipher
+      cipher.upcase.should eq cipher
```

Comment:
I'm not entirely sure about this, maybe checking for not empty would be sufficient.

Reply:
The best would be to check if the cipher is one of https://github.com/openssl/openssl/blob/master/include/openssl/ssl.h#L176, but this was complicating the test.